scripts/deploy.sh

```diff
@@ -71,15 +81,21 @@ elif [[ ! -z "$dtag" && -z "$ptag" && -z "$rtag" ]]; then
         exit 1
     fi
 
-    # tag it locally
+    # Merge branch into master and push to origin
+    git checkout master
+    git pull origin
+    git merge --no-ff release_$rtag -m "Merge branch devel_$rtag"
+    git push origin master
+    git branch -d release_$rtag
+
```

Comment:
@bryevdv we talked about adding this code into creating dev versions, but it seems like it's doing too much. I kind of feel like all we need to do here is call `version_update.py` to update the version strings.